From: Misael Lopez Cruz <misael.lopez@ti.com>
Date: Sat, 29 Jan 2011 18:16:42 +0000 (-0600)
Subject: ASoC: twl6040: Export codec's bias level setting
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=e3250f599279352438a46b30ac24eaec298d34da

ASoC: twl6040: Export codec's bias level setting

Export TWL6040 bias level as it maybe required to change out of
regular DAPM bias level setting, for example during DAI link
enable or disable.

Change-Id: If83cb3d423806fc7fce3e343da9f46bba78cd771
Signed-off-by: Misael Lopez Cruz <misael.lopez@ti.com>
---

Index: kernel/sound/soc/codecs/twl6040.c
===================================================================
--- kernel.orig/sound/soc/codecs/twl6040.c	2011-04-29 12:22:34.199605465 +0900
+++ kernel/sound/soc/codecs/twl6040.c	2011-04-29 13:05:01.139645579 +0900
@@ -865,7 +865,7 @@
 	return 0;
 }
 
-static int twl6040_set_bias_level(struct snd_soc_codec *codec,
+int twl6040_set_bias_level(struct snd_soc_codec *codec,
 				enum snd_soc_bias_level level)
 {
 	struct twl6040_codec *twl6040 = codec->control_data;
@@ -902,6 +902,7 @@
 
 	return 0;
 }
+EXPORT_SYMBOL(twl6040_set_bias_level);
 
 /* set of rates for each pll: low-power and high-performance */
 
Index: kernel/sound/soc/codecs/twl6040.h
===================================================================
--- kernel.orig/sound/soc/codecs/twl6040.h	2011-04-29 12:22:34.199605465 +0900
+++ kernel/sound/soc/codecs/twl6040.h	2011-04-29 13:05:01.139645579 +0900
@@ -29,8 +29,7 @@
 
 void twl6040_hs_jack_detect(struct snd_soc_codec *codec,
 			    struct snd_soc_jack *jack, int report);
-
-static int twl6040_set_bias_level(struct snd_soc_codec *codec,
-				enum snd_soc_bias_level level);
+int twl6040_set_bias_level(struct snd_soc_codec *codec,
+			   enum snd_soc_bias_level level);
 
 #endif /* End of __TWL6040_H__ */
